### PR TITLE
ticket 5 producing results

### PR DESCRIPTION
**Pull request by bigsegs**
I 'think' this is right if 'votes' is the total number of votes on all comments forEach article??

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -96,7 +96,9 @@ describe("GET /api/topics ", () => {
       .then((result)=>{
         expect(result.body).toHaveProperty('articles');
         expect(Array.isArray(result.body.articles)).toBe(true);
-        expect(result.body.articles).toBeSortedBy("created_at",{descending:true});
+        expect(result.body.articles.length).toBe(12)
```
Comment (JVal-NC) [inline __tests__/app.test.js:107]: Good tests, would be great to see the property data types asserted against. Same for test on line 54.

File: articleModel.js [hunk 2]
```diff
@@ -29,44 +28,32 @@ exports.selectArticleById=(id)=>{
 }
 
 exports.selectAllArticles=()=>{
-  const  queryString=`SELECT article_id,title,topic,A.author,
-  created_at,votes,
-   U.avatar_url AS article_img_url
+  const  queryString=`SELECT A.article_id,A.title,A.topic,
```
Comment (JVal-NC) [inline articleModel.js:31]: Chris, looks really good. I would question if we need the dot notation referring to articles as 'A'?

**Review by JVal-NC — COMMENTED** [review]
Strong work so far Chris, couple things to consider in the comments. Another PR when your ready please.

**Review by pippaamy — APPROVED** [review]
great changes, lets get this merged!

**Pull request merged**